registry/tests/access_test.py

```diff
@@ -448,6 +449,16 @@ def testRemovePublicNoPayments(self):
         can_access = data.get('users')
         assert can_access == [self.user]
 
+    def testShareTeamFails(self):
+        resp = self._share_package(self.user, self.pkg, 'team')
+        assert resp.status_code == requests.codes.forbidden
+
+    @patch('quilt_server.views.ALLOW_PUBLIC_USERS', False)
+    @patch('quilt_server.views.ALLOW_TEAM_USERS', True)
```

Comment:
we only ever patch `(ALLOW_PUBLIC_USERS, ALLOW_TEAM_USERS) = (False, True)`. i know that `(True, True)` is not possible, but maybe we should at least have an xfail test case. i assume `(True, False)` is already handled by legacy tests.